### ISSUE TITLE
tBTC dApp Updated with Latest Brand VIS

### ISSUE THREAD

**Issue by creckhow** [keep-network/tbtc-dapp#217]

**Comment by Gluzman** [comment]
@ironng so i'm not sure how else to bring the dapp closer to the new site design without the total UI redesign, so these are some color swaps to get it started:

Figma: https://www.figma.com/file/wSIaVX0Dh9IJDGMrfGZILd/tBTC-dApp?node-id=0%3A1


![Pay BTC](https://user-images.githubusercontent.com/7621557/84522850-58588900-aca5-11ea-8ea4-b9e231044266.png)

![begin dApp](https://user-images.githubusercontent.com/7621557/84522845-57bff280-aca5-11ea-8577-2babd15a04ea.png)

![Interim invoice loading page](https://user-images.githubusercontent.com/7621557/84522846-58588900-aca5-11ea-9acf-94bbe8916f9b.png)


the sand dollar "waiting animation" is awkward, but here's some options:
- 1. use same color as it is currently.
- 2. just do this black and dark grey, and it'll be more subtle than color.
- 3. very slow rotatation animation of the sand dollar (when there is not currency icon inside).

**Comment by ironng** [comment]
@Gluzman I think the color swaps do help a lot. For the "waiting animation" I think options 2 or 3 sound good to me.

**Comment by Gluzman** [comment]
@ironng do you need an export of the sand dollar SVG for anything or are you too to grab what you need from Figma? Is the rotation, for example, something you’d need from me or you can do?

**Comment by ironng** [comment]
@Gluzman I'll check out the existing svg/what's in Figma and see how they're structured and let you know. Thanks for verifying!

**Comment by Shadowfiend** [comment]
One question here as we do this: do we still need what are currently the hackiest warning banners in the world?

**Comment by mhluongo** [comment]
We do, but I'd love for them to not be so hacky 😂

**Comment by ironng** [comment]
@Gluzman What do you think of this wip animation that covers options 2 + 3 in this [flowdock thread](https://www.flowdock.com/app/cardforcoin/design/threads/_yt9wog0Hxb2iVUAa5ybkWH-AH4)? (Posted to FD since GH doesn't support video uploads)

**Comment by ironng** [comment]
@Gluzman does this issue also encompass updates to the dApp landing page (the one that looks like the old marketing site)?  I didn't see anything in the Figma file. For now I can free-hand it a bit and do the color swaps there too, but I might need a bit more guidance for the footer. Off the top of my head, we can remove the swirly background image and update the newsletter form style look a bit closer to the marketing site?  (Or the dApp landing page is a separate issue, and we'll update it later, let me know.)

Also, I noticed in the new footer that we dropped the tbtc logo and about link, but do we want to keep it for the landing page? (There is also a new PR open (#200) to add the faq link.)
